lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -109,20 +129,31 @@ def num_lines
                                     .map { |l| (l.length / max_width).ceil }
                                     .reduce(&:+)
```

Comment:
Filtering on a substring that does not exist leads to an empty options array, for which this `reduce` returns `nil` leading to a no-method error for `total_non_empty_lines +...` below.